components/base/Typography.tsx

```diff
@@ -136,6 +137,50 @@ export const listStyles = css`
   }
 `;
 
+// Table styles
+export const tableStyles = css`
```

Comment:
I noticed that the tables do not overflow gracefully on smaller screens ([example](https://deploy-preview-63--dwolla-dev-portal.netlify.app/docs/guides/migrate-to-v2/05-v1-to-v2-endpoint-changes#v1-to-access-api-endpoint-changes))